README.md

```diff
@@ -43,4 +42,119 @@ The JVM will then listen for a debugger to be attached on port 8189.  The JVM wi
 the debugger to be attached before starting Echo; the relevant JVM arguments can be seen and
 modified as needed in `build.gradle`.
 
-[//]: # "Only here to retrigger the echo build"
+## Configuration
+`echo` can run in two modes: **in-memory** and **SQL**.
+
+**In-memory** mode keeps all CRON trigger information in RAM.  
+While this is simpler to configure (this is the default) the in-memory mode does not provide for any redundancy because it requires that a single instance of `echo` scheduler be running. If there are multiple instances, they will all attempt to start executions for a given CRON trigger. There is no locking, leader election, or any other kind of coordination between scheduler instances using the in-memory mode.  
+If/when this single instance goes down, CRON triggers will not fire.
+
+**SQL** mode keeps all CRON trigger information in a single SQL database. This allows for multiple `echo` scheduler instances to run providing redundancy (only one instance will trigger a given CRON).
+
+To run in SQL mode you will need to initialize the database and provide a connection string in `echo.yml` (note these instructions assume MySQL).
+1. Create a database.
+2. Initialize the database by running the script (MySQL dialect) provided [here](echo-scheduler/src/main/resources/db/database-mysql.sql)
+3. Configure the SQL mode in `echo.yml` (obviously, change the connection strings below):
+    ```yaml
+    sql:
+    enabled: true
```

Comment:
This and following lines should be indented one more level

Reply:
thanks!